doc/findinfiles.rst

```diff
@@ -62,8 +62,9 @@ Finally, to change the number of matches displayed, select the :guilabel:`Set ma
    :alt: Spyder Find pane showing window with maximum results dialog open
 
 
+==================
 Related components
-~~~~~~~~~~~~~~~~~~
+==================
```

Comment:
This will cause a merge conflict on @ccordoba12 's other PR, which does the same thing (except it also adds another blank line above, as is required for h2 headings as this now is. I would suggest undoing the change here by amending your commit (undo the change or check out the file from the previous commit, and then do `git commit -a --amend --no-edit && git push -f`. To avoid the need for a rebase, I'll avoid applying any suggestions here until you've gone ahead with that (or decided otherwise with @ccordoba12 ).